packages/editor/src/store/effects/posts.js

```diff
@@ -59,6 +57,7 @@ export const requestPostUpdate = async ( action, store ) => {
 	}
 
 	let edits = getPostEdits( state );
+	const isAutosave = !! action.options.isAutosave;
```

Comment:
Why bother assigning this when it looks like you're just using it once, immediately below? `action.options.isAutosave` isn't that much longer, and I wonder why you're checking if it's truthy when it already looks like a boolean.

Reply:
It's actually used in two places.

Reply:
Oops, I guess I missed it! My bad 😓